### ISSUE TITLE
[AIR output] UX issues of new AIR output for tune_torch_benchmark.py in release test (AIR_VERBOSITY=0, 1, 2)

### ISSUE THREAD

**Issue by scottsun94** [ray-project/ray#33847]
#### AIR_VERBOSITY = 1
1. Didn't see the configurations for the experiment.
2. Saw some weird duplication in the first trial table
```
Trial status: 1 RUNNING | 7 PENDING
Current time: 2023-03-28 19:11:40 (running for 00:00:00.07)
Logical resource usage: 8.0/64 CPUs, 0/0 GPUs
Trial name                status
------------------------  --------
TorchTrainer_0b45c_00000  RUNNING
Trial name                status
------------------------  --------
TorchTrainer_0b45c_00000  RUNNING
TorchTrainer_0b45c_00001  PENDING
TorchTrainer_0b45c_00002  PENDING
TorchTrainer_0b45c_00003  PENDING
TorchTrainer_0b45c_00004  PENDING
TorchTrainer_0b45c_00005  PENDING
TorchTrainer_0b45c_00006  PENDING
TorchTrainer_0b45c_00007  PENDING
```

3. Regarding the format of the trial table, can we just use 1 single line as a divider?
```
Trial name                status      iter    total time (s)      loss    local_time_taken
-------------------------------------------------------------------------------------------
TorchTrainer_0b45c_00000  RUNNING        2           26.0664  0.841929             22.4405
```

4. We probably should be more explicit about what the time refers to and render time in a more readable format.
```
# current implementation
Current time: 2023-03-28 19:11:40 (running for 00:01:39.40)

# suggestion
Current time: 2023-03-28 19:11:40. Total running time: 1min, 39.40s
```

5. Similar to the suggestion 4 here: https://github.com/ray-project/ray/issues/33810#issue-1644408299, we can simplify the output?

```
# current implementation
Trial TorchTrainer_0b45c_00006 (20 iters) finished at 2023-03-28 19:15:25 (running for 00:03:44.61)
local_time_taken: 209.88436901900002
loss: 1.795833697744236
time_this_iter_s: 10.39311170578003
time_total_s: 214.88134670257568
training_iteration: 20

# simplified output
Trial TorchTrainer_0b45c_00006 (20 iterations) finished at 2023-03-28 19:15:25. Total training time: 3min 24.29s. Total running time: 3min 44.61s
-------------------------
loss: 1.79583
-------------------------
```

6. When running the script with `ENABLE_RICH=1`
- not sure if the line " 🌟 Ray Tune Trial Status Table 🌟 " is needed
- For the table, maybe we could just `use box=box.SQUARE`. Something like this?
- let's use lower cases for all the headers for consistency?
- +1 to Kai's point that time format should be rendered in a more user-friendly way and keep fewer decimal places.
<img width="658" alt="Screen Shot 2023-03-28 at 9 40 44 PM" src="https://user-images.githubusercontent.com/9677264/228428670-7f6fd2b5-7c22-4d57-9a91-8f61fb115171.png">

#### AIR_VERBOSITY = 2

1. How about changing the frequency of heartbeat to be: every 30 seconds since the last result reporting?
- in the following example, the iteration results are reported pretty frequently. It feels less necessary to report heartbeat in this case. 

```
2023-03-29 09:43:32,235 WARNING tune.py:554 -- Testing new AIR console output flow with verbosity=2. This will also disable the old flow - setting it to 0 now.
Training started with configuration:
train_loop_config:
  batch_size: 64
  epochs: 20
  lr: 0.001

Training on iteration 1. Current time: 2023-03-29 09:43:35 (running for 00:00:03.44)

(omitted user logs)

Training finished iter 1 at 2023-03-29 09:43:52 (running for 00:00:19.62)
local_time_taken: 11.469622267000005
loss: 2.233998278903354
time_this_iter_s: 16.178134441375732
time_total_s: 16.178134441375732
training_iteration: 1

(omitted user logs)

Training finished iter 2 at 2023-03-29 09:44:03 (running for 00:00:30.93)
local_time_taken: 22.815137599000025
loss: 2.1184383942063447
time_this_iter_s: 11.307494878768921
time_total_s: 27.485629320144653
training_iteration: 2

(omitted user logs)

Training on iteration 3. Current time: 2023-03-29 09:44:08 (running for 00:00:35.94)

(omitted user logs)

Training finished iter 3 at 2023-03-29 09:44:14 (running for 00:00:42.12)
local_time_taken: 33.996853163000026
loss: 1.9640579383084729
time_this_iter_s: 11.185982465744019
time_total_s: 38.67161178588867
training_iteration: 3
```

2. similar to comments in other issues, let's switch to "iteration" instead of "iter" when reporting results since we don't need to save space here.
```
# current implementation
Trial TorchTrainer_669b2_00000 finished iter 20 at 2023-03-29 09:51:32 (running for 00:04:00.08)

# suggestion
Trial TorchTrainer_669b2_00000 finished iteration 20 at 2023-03-29 09:51:32. Training time for this iteration:  13.08s
```

3. When the terminal height is smaller than the sticky rich table, the table will get truncated and users cannot see the rest of it. They have to adjust the height.
I'm not sure if there is a good solution to this issue. This doesn't seem an important one to fix for now.
<img width="896" alt="Screen Shot 2023-03-29 at 10 17 22 AM" src="https://user-images.githubusercontent.com/9677264/228617858-9ff0acd4-21d2-4eb3-b2b0-07a80399bc60.png">

### Versions / Dependencies

nightly

### Reproduction script

python workloads/tune_torch_benchmark.py --num-trials 8 --num-workers 4
Link to the output: https://anyscaleteam.slack.com/files/U02837M6JQ7/F050RBSE0DR/tune_torch_benchmark.py

### Issue Severity

Low: It annoys or frustrates me.

**Comment by xwjiang2010** [comment]
Issue 2 is fixed in PR: https://github.com/ray-project/ray/pull/33767
However we are not allowed to land anymore.

> AIR_VERBOSITY = 2
How about changing the frequency of heartbeat to be: every 30 seconds since the last result reporting?
in the following example, the iteration results are reported pretty frequently. It feels less necessary to report heartbeat in this case.

In our last design discussion `AIR_VERBOSITY=2` for Tuning just means we are gonna print out per trial result as they come in. This is not a "heartbeat" concept per se. The reason you feel it's frequent is because these results take less time to compute, which is not controlled by us.

**Comment by scottsun94** [comment]
> Issue 2 is fixed in PR: #33767 However we are not allowed to land anymore.
> 
> > AIR_VERBOSITY = 2
> > How about changing the frequency of heartbeat to be: every 30 seconds since the last result reporting?
> > in the following example, the iteration results are reported pretty frequently. It feels less necessary to report heartbeat in this case.
> 
> In our last design discussion `AIR_VERBOSITY=2` for Tuning just means we are gonna print out per trial result as they come in. This is not a "heartbeat" concept per se. The reason you feel it's frequent is because these results take less time to compute, which is not controlled by us.

See my example above. It's a train-only example. I'm suggesting it for train-only flow.
For tune, it's fine to keep it as is.

**Comment by scottsun94** [comment]
Created separate issues for unsolved ones.
